### PR TITLE
Add support for schema without proxy support

### PR DESCRIPTION
**Pull request by witekest**
Add migration procedure for older schema without keys for exporters proxy
configuration.

### CODE REVIEW

File: prometheus-exporters-formula/prometheus-exporters-formula.spec [hunk 1]
```diff
@@ -18,13 +18,15 @@
...
 Name:           prometheus-exporters-formula
-Version:        1.0.0
+Version:        1.0.1
 Release:        0
 Summary:        Salt formula for installing and configuring Prometheus exporters
 License:        Apache-2.0
 Group:          System/Packages
 Url:            https://github.com/SUSE/salt-formulas
 Source:         prometheus-exporters-formula-%{version}.tar.gz
+BuildRequires:  python3
```
Comment (renner) [inline prometheus-exporters-formula/prometheus-exporters-formula.spec:28]: I might not be completely in the picture about our needs at build time here, but shouldn't runtime actually be enough (`Requires`)?

File: prometheus-exporters-formula/prometheus-exporters-formula.changes [hunk 2]
```diff
@@ -1,3 +1,5 @@
+- Add support for older schema without proxy configuration
+
```
Comment (renner) [inline prometheus-exporters-formula/prometheus-exporters-formula.changes:2]: Here we will likely need a new date header (like the one below), before submitting it to OBS?

**Review by renner — APPROVED** [review]
The code looks good to me, only some minor comments about the spec file and changes.

File: prometheus-exporters-formula/prometheus-exporters-formula.spec (hunk 1 above)
Comment (witekest) [reply prometheus-exporters-formula/prometheus-exporters-formula.spec:28]: At IBS some tests where failing without this. If I recall correctly it was `50-check-installtest`.

File: prometheus-exporters-formula/prometheus-exporters-formula.changes (hunk 2 above)
Comment (witekest) [reply prometheus-exporters-formula/prometheus-exporters-formula.changes:6]: Done

**Pull request merged**